Gem/Code/Source/CullingAndLodExampleComponent.cpp

```diff
@@ -233,8 +233,8 @@ namespace AtomSampleViewer
             dirLightFP->SetViewFrustumCorrectionEnabled(handle, m_isCascadeCorrectionEnabled);
             dirLightFP->SetShadowFilterMethod(handle, s_shadowFilterMethods[m_shadowFilterMethodIndex]);
             dirLightFP->SetShadowBoundaryWidth(handle, m_boundaryWidth);
-            dirLightFP->SetPredictionSampleCount(handle, m_predictionSampleCount);
-            dirLightFP->SetFilteringSampleCount(handle, m_filteringSampleCount);
+            dirLightFP->SetPredictionSampleCount(handle, static_cast<uint16_t>(m_predictionSampleCount));
```

Comment:
`m_predictionSampleCount` and `m_filteringSampleCount` definitely should be just `uint16_t` originally I think. The reason they aren't is because later code uses `ImGUI::SliderInt`. Instead of calling `ImGUI::SliderInt`, we should just call `ImGUI::SliderScalar` (and pass `ImGuiDataType_U16` as the second argument, all other arguments the same)